### PR TITLE
Supprt layout inference for Operations

### PR DESCRIPTION
**Pull request by liyuenan2333**
Signed-off-by: yuenan.li <yuenan.li@verisilicon.com>

### CODE REVIEW

File: src/tim/vx/operation.cc [hunk 1]
```diff
@@ -43,6 +43,7 @@ OperationImpl::OperationImpl(Graph* graph, uint32_t operation_id,
                            output_cnt_, NULL)) {
   SetRoundingPolicy();
   node_->uid = graph_->graph()->cur_nid;
+  node_->attr.enable_op_constraint_check = false;
```
Comment (thezha) [inline src/tim/vx/operation.cc:46]: Why do we disable constraint check? If we want to do this, let's move this into a separate patch.

Comment (liyuenan2333) [reply src/tim/vx/operation.cc]: got it! I've remove it from this patch.
